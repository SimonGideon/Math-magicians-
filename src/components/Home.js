import React from 'react';
import '../App.css';

const Home = () => (
  <div className="HomeConctainer">
    <h1>Welcome to our home Page</h1>
    <div className="HomeBody">
      <p>
        Are you a big fan of Maths? Do numbers excite you? Then we suggest you
        cling on to this article because here we have provided some Maths
        amazing facts and wonders for you that can increase your knowledge on
        this subject. You can also help your children learn about these
        interesting facts about Maths. Who can say maybe he will find a whole
        new enthusiasm to learn Maths?
      </p>
      If you want to increase your childâ€™s interest in this particular subject,
      you might not face many difficulties because the best part is that you do
      not need to make him solve some serious Maths problems to learn them. So,
      to help you and your kids, we have come up with some interesting, fun, and
      amazing facts about Maths that you may find very enlightening.
      <p />
    </div>
  </div>
);

export default Home;
