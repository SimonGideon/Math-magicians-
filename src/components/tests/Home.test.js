import renderer from 'react-test-renderer';
import Home from '../Home';

it('Renders works', () => {
  const tree = renderer.create(<Home />).toJSON;
  expect(tree).toMatchSnapshot();
});
