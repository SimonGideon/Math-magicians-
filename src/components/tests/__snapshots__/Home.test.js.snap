// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders works 1`] = `[Function]`;
